.changeset/angry-timers-press.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-cloud-sdk/connectivity': patch
+'@sap-cloud-sdk/mail-client': patch
+---
+
+[Fixed Issues] Set `_readableListening` property of Socket to `true` to allow nodemailer to receive SMTP greeting messages. Fixes `Greeting never received` error when sending emails to onPrem mail servers.
```

Comment:
Maybe mention what was fixed and then how it was done. Starting with the technical details might not be relevant for most users.
```suggestion
[Fixed Issues] Set `_readableListening` property of socket to `true` to allow nodemailer to receive SMTP greeting messages. Fixes `Greeting never received` error when sending emails to On-Premise mail servers.
```